src/system/components/sente.cljc

```diff
@@ -67,7 +67,12 @@
          (sente/chsk-disconnect! chsk))
        (when-let [stop-f router]
          (stop-f))
-       (dissoc component :router :chsk :ch-recv :chsk-send! :chsk-state))))
+       (assoc component
+              :router nil
+              :chsk nil
+              :ch-recv nil
```

Comment:
Yes. Good catch. Thanks!

When you have time, could you please squash the commits so that I can proceed with merging. Thank you.